cmd/fleet/handleCheckin.go

```diff
@@ -255,12 +256,19 @@ func (ct *CheckinT) resolveSeqNo(ctx context.Context, req CheckinRequest, agent
 func (ct *CheckinT) fetchAgentPendingActions(ctx context.Context, seqno sqn.SeqNo, agentId string) ([]model.Action, error) {
 	now := time.Now().UTC().Format(time.RFC3339)
 
-	return dl.FindActions(ctx, ct.bulker, dl.QueryAgentActions, map[string]interface{}{
+	actions, err := dl.FindActions(ctx, ct.bulker, dl.QueryAgentActions, map[string]interface{}{
 		dl.FieldSeqNo:      seqno.Get(0),
 		dl.FieldMaxSeqNo:   ct.gcp.GetCheckpoint(),
 		dl.FieldExpiration: now,
 		dl.FieldAgents:     []string{agentId},
 	})
+	if err != nil {
+		if errors.Is(err, es.ErrIndexNotFound) {
```

Comment:
why? this check is already in findActions
https://github.com/elastic/fleet-server/blob/master/internal/pkg/dl/actions.go#L89

Reply:
hmm maybe I had an outdated master when I was testing, will remove and test before merging